api/envoy/extensions/request_id/uuid/v3/uuid.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package envoy.extensions.request_id.uuid.v3;
+
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.request_id.uuid.v3";
+option java_outer_classname = "UuidProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: UUID]
+// [#extension: envoy.request_id.uuid]
+
+// Configuration for the default UUID request ID extension which has the following behavior:
+//
+// 1. Request ID is propagated using the *x-request-id* header.
+//
+// 2. Request ID is a universally unique identifier (UUID).
+//
+// 3. Tracing decision (sampled, forced, etc) is set in 14th byte of the UUID. By default this will
+//    overwrite existing UUIDs received in the *x-request-id* header if the trace sampling decision
+//    is changed.
```

Comment:
Is there any documentation on the format for the 14th byte to link to here? Is this just 1 or 0? Presumably the overwrite only happens if we are deciding to sample on a request ID that doesn't already indicate, vs. the other way around? Sorry if these aren't great questions, but as an unfamiliar reader, this is likely a question to arise.

Reply:
I don't think there is but I can beef up the docs for this.